src/main/java/survey/Survey.java

```diff
@@ -23,13 +26,17 @@
     /*
      * A default constructor
      */
-    public Survey(){ }
+    public Survey(){
+        this(null);
+    }
 
     /*
      * A constructor
      * @param name {String}
      */
-    public Survey(String name){ this.name = name; }
+    public Survey(String name){
+        this(name, null);
```

Comment:
I added constructor chaining because in all cases I need to generate a UUID, which is done in the constructor below.